src/libcore/num/mod.rs

```diff
@@ -1278,15 +1275,9 @@ macro_rules! uint_impl {
             }
         }
 
-        /// Wrapping (modular) division. Computes `self / other`,
-        /// wrapping around at the boundary of the type.
-        ///
-        /// The only case where such wrapping can occur is when one
-        /// divides `MIN / -1` on a signed type (where `MIN` is the
-        /// negative minimal value for the type); this is equivalent
-        /// to `-MIN`, a positive value that is too large to represent
-        /// in the type. In such a case, this function returns `MIN`
-        /// itself.
+        /// Wrapping (modular) division. Computes `self / other`.
+        /// This operation only makes sense on signed types.
+        /// It is here for completeness only.
```

Comment:
Yea, I should probably word that differently. Wrapped division on unsigned types is just normal division. There's no way wrapping could ever happen. So this function is here, so all operations are accounted for in the wrapping operations.